File: tourneyClient/src/models/entities/Characters/Sora.ts
import { CharacterId } from "@/models/Enums/CharacterId";
import { Character } from "../Character"
import { CharacterName } from "@/models/Enums/CharacterName";
import { Archetype } from "@/models/Enums/Archetype";
import { FallSpeed } from "@/models/Enums/FallSpeed";
import { WeightClass } from "@/models/Enums/WeightClass";
import { TierPlacement } from "@/models/Enums/TierPlacement";

export const Sora: Character =
{
    id: CharacterId.Sora,
    characterName: CharacterName.SORA,
    archetype: Archetype.MIX_UP,
    fallSpeed: FallSpeed.FLOATY,
    weightClass: WeightClass.LIGHTWEIGHT,
    tierPlacement: TierPlacement.A
}

export default Sora; 

File: tourneyAPI/Models/Entities/Characters/Sora.cs
namespace Entities;

using Enums;

// Defines the competitive profile metadata for this playable character.
public class Sora : Character
{
    // Initializes this character's default competitive attributes for matchmaking and tier logic.
    public Sora()
    {
        Id = CharacterId.Sora;
        characterName = CharacterName.SORA;
        archetype = Archetype.MIX_UP;
        fallSpeed = FallSpeed.FLOATY;
        weightClass = WeightClass.LIGHTWEIGHT;
        tierPlacement = TierPlacement.A;
    }

}